### ISSUE TITLE
Property "$t" was accessed during render but is not defined on instance

### ISSUE THREAD

**Issue by agracia-foticos** [nuxt/nuxt.js#15269]
### Environment

Nuxt project info:                                                                                                 10:20:13

------------------------------
- Operating System: `Linux`
- Node Version:     `v16.18.0`
- Nuxt Version:     `3.0.0-rc.12`
- Nitro Version:    `0.6.0`
- Package Manager:  `npm@8.19.2`
- Builder:          `vite`
- User Config:      `srcDir`, `modules`, `compilerOptions`, `vue`, `vite`, `telemetry`, `algolia`, `head`, `auth`, `css`, `build`, `runtimeConfig`
- Runtime Modules:  `genesis-design-system/nuxt`, `@nuxtjs/tailwindcss@5.3.5`, `@nuxtjs/algolia@1.1.4`, `@nuxtjs-alt/auth@2.0.20`, `@nuxtjs/device@3.0.0`, `unplugin-icons/nuxt`, `@pinia/nuxt@0.4.3`, `@nuxtjs/i18n@8.0.0-alpha.3`
- Build Modules:    `-`
------------------------------

### Reproduction

I am doing a crawl of the website and randomly it is throwing me this error in the server logs

[Vue warn]: Property "$t" was accessed during render but is not defined on instance.
[nitro] [dev] [unhandledRejection] TypeError: _ctx.$t is not a function
    at _sfc_ssrRender (/var/www/_next/src/pages/t/[slug].vue:152:239)
    at renderComponentSubTree (/var/www/_next/node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:254:17)
    at /var/www/_next/node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:185:29
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)

### Describe the bug

I am doing a crawl of the website and randomly it is throwing me this error in the server logs

[Vue warn]: Property "$t" was accessed during render but is not defined on instance.
[nitro] [dev] [unhandledRejection] TypeError: _ctx.$t is not a function
    at _sfc_ssrRender (/var/www/_next/src/pages/t/[slug].vue:152:239)
    at renderComponentSubTree (/var/www/_next/node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:254:17)
    at /var/www/_next/node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:185:29
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)

### Logs

```shell
[Vue warn]: Property "$t" was accessed during render but is not defined on instance.
[nitro] [dev] [unhandledRejection] TypeError: _ctx.$t is not a function
    at _sfc_ssrRender (/var/www/_next/src/pages/t/[slug].vue:152:239)
    at renderComponentSubTree (/var/www/_next/node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:254:17)
    at /var/www/_next/node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:185:29
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Comment by carbdias** [comment]
I have the same error, with a basic Nuxt i18n next configuration. Returning an async locales, when try to access the object with $t, it says the same error.

**Comment by manniL** [comment]
@carbdias Create a reproduction and an issue in https://github.com/nuxt-modules/i18n

**Comment by carbdias** [comment]
Ok, thank, I've managed to solve it

**Comment by anrypwnz** [comment]
> Ok, thank, I've managed to solve it

How? Describe please

**Comment by carbdias** [comment]
I've added the await key word before calling the translation function in the i18n.config file, where the node messages exist. Also, for that await work at the config level (when compiling) it was necessary to install a plugin called vite-plugin-top-level-await, so that vite accepts this sintaxis here. Reading the JSON files were done with await import(...), where before I had require().
But in the end when I deployed the application, because the agent had a higher node version, it wasn't accepting this plugin, and I had to undo what I've done regarding aync translations. I've made all of them async, and Importing the JSON files with "import file_en from '..'' sintaxis.
